### PR TITLE
Website Fixes

### PR DESCRIPTION
**Pull request by danielogen**
#### Summary of Changes (What does this PR do?)
- Correct blog title on browser tab
- Display all the blogs in a grid layout 
-
#### Is this change ready to hit production in its current state?
- [X] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [X] I consider this code done
- [X] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [PLAT-583](https://airqoteam.atlassian.net/browse/PLAT-583)

#### Screenshots (optional)
Screen Shot 2021-08-31 at 12 30 30 PM

### CODE REVIEW

File: web/application/views/airqo-blog.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <style>
-.slider-post[data-v-5e6923b3] {
+/* .grid-post[data-v-5e6923b3] {
     position: relative;
```
Comment (Baalmart) [inline web/application/views/airqo-blog.php:3]: Thanks, awesome stuff @danielogen 
Could we remove these comments?
Unless we plan to use them in the near future?

File: web/application/controllers/Airqo.php [hunk 2]
```diff
@@ -57,35 +57,35 @@ public function projects()
...
-			$data['posts'] = $this->AirqoModel->get_post_categories($category);
-			$data['suggestions'] = $this->AirqoModel->get_home_random();
-			$data['categories'] = $this->AirqoModel->get_categories();
-
-			$this->load->view('lib/header',$data);
-			$this->load->view('airqo-blog-new',$data);
-			$this->load->view('lib/footer',$data);
-		}
-	}
+	// public function blog($category = null)
```
Comment (Baalmart) [inline web/application/controllers/Airqo.php:60]: Could we remove/delete these comments?
Unless we plan to use them in the near future?

**Review by Baalmart — COMMENTED** [review]
LGTM, thanks @danielogen !
Just left a few comments for review.

File: web/application/views/airqo-blog.php (hunk 1 above)
Comment (danielogen) [reply web/application/views/airqo-blog.php]: resolved

File: web/application/controllers/Airqo.php (hunk 2 above)
Comment (danielogen) [reply web/application/controllers/Airqo.php]: resolved

**Review by NoahNsimbe — APPROVED** [review]
LGTM!

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**